Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM ultimaker/cura-build-environment:1
+
+# Environment vars for easy configuration
+ENV CURA_APP_DIR=/srv/cura
+
+# Ensure our sources dir exists
+RUN mkdir $CURA_APP_DIR
+
+# Setup CuraEngine
+ENV CURA_ENGINE_BRANCH=master
```

Comment:
And all the `ENV`s can be compressed into a single layer like this:
```
ENV A=a \
    B=b \
    ...
```

Reply:
The reason I didn't do that is because if you change them, it will rebuild the whole image (e.g. recompile CuraEngine), even when that is not necessary.